__tests__/app.test.js

```diff
@@ -326,4 +326,22 @@ describe("POST /api/articles/:article_id/comments", () => {
         });
```

Comment:
I think there's two tests missing, but feel free to correct me. 

- [ ] 400 - invalid article id
- [ ] 400 missing required field, on the post object